config/settings/segmentedtickerplant.q

```diff
@@ -15,8 +15,8 @@ loadprocesscode:1b
 
 \d .eodtime
 
-datatimezone:`GMT
-rolltimezone:`GMT
+datatimezone:@[value;`datatimezone;`GMT];
```

Comment:
I don't think either of these definitions are required in here. They should just be inherited from the eodtime code. Can you remove these and re-run the tests?

Reply:
Sure, similar [definitions are defined on FSP](https://github.com/AquaQAnalytics/TorQ-Finance-Starter-Pack/blob/master/appconfig/settings/segmentedtickerplant.q#L20), should I submit a PR to remove these also? So that these options are handled solely by config `default.q`.